analyze.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+ARTIFACT_DIR=${ARTIFACT_DIR:=/tmp/artifacts}
```

Comment:
@spadgett do you need to set this again? Isn't this already specified when ci calls the script?

`  commands: export ARTIFACT_DIR=/tmp/artifacts; ./analyze.sh`

Reply:
It will use the value from the environment. This defaults it if not set.